source/features/pr-approvals-count.css

```diff
@@ -1,19 +1,16 @@
 /* Show approvals count in PR list */
-.js-issue-row .text-gray [aria-label*='approval'], /* GHE */
-.js-issue-row .color-text-secondary [aria-label*='approval'] {
+.js-issue-row :is(.text-gray, .color-text-secondary) [aria-label*='approval'] {
```

Comment:
No GHE merging

Reply:
its the .text-gray

Reply:
I explained this in a previous review: GHE-specific selectors are meant to be removed and, like the “repository refresh” changes, it’s easier to keep them on their own instead of intermixing the code. When we finally drop the selectors, we just delete the whole line.